### ISSUE TITLE
Switch to issue templates to new issue form templates

### ISSUE THREAD

**Issue by fdellwing** [RocketChat/feature-requests#483]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
GitHub has a new feature for issue templates that allow you to use complex forms with required fields and much more. You should switch to them to really force the users to create usable issue reports.

**Describe alternatives you've considered**
None

**Additional context**
Take a look at https://github.com/ohmyzsh/ohmyzsh/pull/10050 to see an implemtation of this.

**Comment by johncrisp** [comment]
Hi and thanks for your feature request and PR.

We did run a bot for a while which was very strict on details, but found it annoyed and upset a lot of people. They opened multiple bugs until such time as it was accepted.

I don't dispute that people should fill in more information where possible. However as we discovered with the bot, you will find that despite being forced to fill in the boxes, people will just type gibberish until it is accepted!!

I am going to review the issue templates as they certainly need rewording and I will look at forms.

**Comment by fdellwing** [comment]
The good thing about the new forms is that you can simply not post the issue if you do not fill every field. That is a lot less frustrating than posting an issue and getting it autoclosed multiple times because you missed some minor detail.

My PR still allows completely blank issues without any enforcements, but it is a lot less prominent

**Comment by johncrisp** [comment]
Indeed, but it doesn't guarantee they won't fill it with gibberish!! That can make it just as tricky to decipher.

I have this on my ToDo list as we know that some of the information and guidance that is provided is not that clear.

By all means find me on open.rocket.chat and talk to me about it.

We'll leave this in abeyance pending a wider review.

**Comment by fdellwing** [comment]
Sure take all the time you need, the feature is still in beta as far as I'm aware, so some details might change.

Just take it as a starting point for what is possible in the future.

**Comment by johncrisp** [comment]
OK, and thanks.
